docs/traceflow-guide.md

```diff
@@ -44,9 +52,10 @@ the Agent configuration:
 
 For antrea-octant-plugin installation, please refer to [antrea-octant-installation](octant-plugin-installation.md).
 
-## Start a New Trace
+## Start a New Traceflow
 
-You can choose to use kubectl together with YAML file, antctl with spec information or Octant UI to start a new trace.
+You can choose to use `kubectl` together with a YAML file, `antctl traceflow`
```

Comment:
*the* `antctl traceflow` command, or *the* Octant UI to start a new trace.

Reply:
Updated.